Lesson_3/home_work_3/index.html

```diff
@@ -0,0 +1,50 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+	<meta name="description" content="Резюме Дробенюка А.Ю.">
+	<meta name="keywords" content="Резюме, Дробенюк, Дробенюк А.ю., Резюме Дробенюка А.Ю.">
+	<meta charset="utf-8">
+	<title>Lesson 3 / Home work</title>
+	<link href="css/style.css" type="text/css" rel="stylesheet">
+</head>
+<body>
+	<img src="img/my_photo.jpg" alt="my_foto">
+	<h1> Дробенюк Андрій Юрійович </h1>
+	<h3>Особиста інформація:</h3>
+	<ul>
+		<li>Дата народження: 23.12.1988 р.</li>
+		<li>Адреса: 33018 м. Рівне, вул. Студентська буд. 7, кв. 59</li>
+		<li>Телефон: <a href="phone:+380965955659">(096) 59 55 659</a></li>
+		<li>Електронна пошта: <a href="mailto:Drobenyuk.A@gmail.com">Drobenyuk.A@gmail.com</a></li>
+	</ul>
+	<h3>Освіта:</h3>
+	<ol>
+		<li>Iз <b>09.2004</b> по <b>06.2008</b> &mdash; ПВНЗ «Рівненський економіко-гуманітарний та інженерний коледж» отримав диплом молодшого спеціаліста за спеціальністю «Обслуговування комп’ютерних та інтелектуальних систем та мереж» (технік-електронник)</li>
+		<li>Iз <b>09.2008</b> по <b>06.2011</b> &mdash; ПВНЗ «Міжнародний економіко-гуманітарний університет ім. ак. Степана Дем’янчука» отримав диплом спеціаліста за спеціальністю «Інформатика» (фахівець з інформатики)</li>
+		<li>Iз <b>02.2016</b> по сьогодні &mdash; Комп'ютерна Академія "Шаг" проходжу курс "Front-end Developer – проектування та розробка інтерфейсів для web-додатків та сайтів".</li>
+	</ol>
+	<h3>Досвід роботи:</h3>
+	<table>
+		<tbody>
```

Comment:
if you have only `tbody`, then you can write without it, browser will put it by default and W3C validator will not rise an error